library/books-api-v2/BookMapper.ts

```diff
@@ -0,0 +1,15 @@
+import { Book as BookV1} from "./v1/Book";
+import { Book } from "./v2/Book";
+
+export class BookMapper {
+    
+    static toV1(book: Book): BookV1 {
```

Comment:
You don't really need a class or `static` here. You can just export the function itself (in a larger project, this would also help with tree-shaking).

```suggestion
/**
 * Converts a V2 Book to a V1 Book.
 */
export function bookToV1(book: Book): BookV1 {
  // ...
}
```

I would consider renaming the file from `BookMapper.ts` to `book-utils.ts` or similar.